ios-base/Networking/APIService.swift

```diff
@@ -74,6 +74,7 @@ open class BaseApiService<T>: APIService where T: TargetType {
   open var jsonDecoder: JSONDecoder {
     let decoder = JSONDecoder()
     decoder.keyDecodingStrategy = .convertFromSnakeCase
```

Comment:
will this take care of `Session. accessToken`?

Reply:
Nope, that's why session still has `CodingKeys`

Reply:
perfect!